### DISCUSSION TITLE
Does this calculation tool support computing weekly-scale sc_PDSI?

### DISCUSSION THREAD

**Discussion by wwingyuu** [monocongo/climate_indices#566] [Q&A]
Does this calculation tool support computing weekly-scale sc_PDSI?

**Comment by monocongo** [comment]
No, we currently only support computation for monthly and daily datasets.

**Comment by monocongo** [comment]
The Palmer indices provided may only be valid for monthly data, I need to dig through the code to confirm this.

**Comment by wwingyuu** [comment]
Thank you very much for your reply. I also wanted to ask, when calculating sd_PDSI, what is the unit of the awc variable in the input?
